api/SWR/CarePackage/useCarePackageApi.js

```diff
@@ -28,7 +28,10 @@ const useCarePackageApi = {
     useGetData(shouldFetch ? `/suppliers${getQueryParamsFromObject({ supplierName })}` : null),
 
   singleSupplier: (supplierId) => useGetData(supplierId ? `/suppliers/${supplierId}` : null, '', {}),
-  singlePackageInfo: (packageId) => useGetData(`${CARE_PACKAGES_URL}/${packageId}`, '', {}),
+  singlePackageInfo: (packageId) => {
+    const response = useGetData(`${CARE_PACKAGES_URL}/${packageId}`, '', {});
+    return { ...response, singlePackageInfoLoading: !!response.data};
```

Comment:
It will be infinite loading if there is an error. Check also on error: `!response.data && !response.error`. Also better to do this in `useGetData` in order to provide `isLoading` property to every request uses `useGetData`